diff --git a/emscripten/tools/shared.orig.py b/emscripten/tools/shared.py
index 455ec00..91330f4 100644
--- a/emscripten/tools/shared.orig.py
+++ b/emscripten/tools/shared.py
@@ -447,20 +447,7 @@ def check_sanity(force=False):
   sanity_file = cache.get_path('sanity.txt')
 
   def sanity_is_correct():
-    if os.path.exists(sanity_file):
-      sanity_data = utils.read_file(sanity_file)
-      if sanity_data == expected:
-        logger.debug(f'sanity file up-to-date: {sanity_file}')
-        # Even if the sanity file is up-to-date we still need to at least
-        # check the llvm version. This comes at no extra performance cost
-        # since the version was already extracted and cached by the
-        # generate_sanity() call above.
-        if force:
-          perform_sanity_checks()
-        else:
-          check_llvm_version()
-        return True # all is well
-    return False
+    return True
 
   if sanity_is_correct():
     # Early return without taking the cache lock
